pilot/pkg/config/memory/store.go

```diff
@@ -161,12 +161,9 @@ func (cr *store) Create(cfg config.Config) (string, error) {
 	_, exists = ns.Load(cfg.Name)
 
 	if !exists {
-		tnow := time.Now()
-		cfg.ResourceVersion = tnow.String()
```

Comment:
The store is meant to simulate k8s store, so it should be setting resource version I think

Reply:
> The store is meant to simulate k8s store, so it should be setting resource version I think

I added the debug log and found that the resourceVersion exists.
When I update the resource, the resourceVersion will change, otherwise it will not change. This is not the same as k8s store.

Reply:
> I added the debug log and found that the resourceVersion exists.

Maybe in current MCP over XDS code - the memory store is used for more than just that though?